Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfiguration.h

```diff
@@ -228,12 +228,12 @@ on the system setting.
     */
 @property (nonatomic) BOOL supportsAdaptiveImageGlyph WK_API_AVAILABLE(macos(15.0), ios(18.0), visionos(2.0));
 
-#if (TARGET_OS_IOS && !TARGET_OS_VISION) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 180000
+#if (TARGET_OS_IOS && !TARGET_OS_VISION) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 180000
```

Comment:
Why do we still need a preprocessor check here? Certainly any SDK we ship in the future will have `__IPHONE_OS_VERSION_MAX_ALLOWED >= 180000` and `__MAC_OS_X_VERSION_MAX_ALLOWED >= 150000`, so this is a no-op.

We shouldn't need anything other than the availability declaration here.

Reply:
We still build WebKit with older SDKS for EWS (until we update to iOS 18 / macOS 15) and downlevels (macOS) right?

In those cases, `UIWritingToolsBehavior` and `NSWritingToolsBehavior` are unavailable.